### ISSUE TITLE
Error: You are not allowed to do this

### ISSUE THREAD

**Issue by NML32** [alejandroliu/pocketmine-plugins#49]
After I add someone to edit the world I loose all my perms in game.
None of the OPs can use WP commands.

**Comment by RocoWayne** [comment]
Yes, i have the same error, dont add a player to auth list of a world. I LOCK the world after edit.

**Comment by xXRyzen** [comment]
Same, cant use /wp command in any world. That feature of disabling commands per world should have came disabled by default. its stupid.
